src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangle.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$mainColor: $white;
+$boxShadow: 0 0 20px 0 rgba($black, 0.3);
```

Comment:
Since you are not reusing that value, should we inline it? Would lead to less indirection...